tests/container_disk_test.go

```diff
@@ -250,17 +250,6 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 
 				By("Checking the writable Image Octal mode")
 				Expect(strings.Trim(writableImageOctalMode, "\n")).To(Equal("640"), "Octal Mode of writable Image should be 640")
-
-				readonlyImageOctalMode, err := tests.ExecuteCommandOnPod(
```

Comment:
Maybe add a test that specifically ensures that we don't try to apply any modifications to the container-disk in such cases?